### ISSUE TITLE
Tag Helper for Bootstrap Panel

### ISSUE THREAD

**Issue by manojkulkarni30** [dpaquette/TagHelperSamples#45]
@dpaquette  I would like contribute tag helper for bootstrap panel. Can i submit a pull request for it ?

Structure:

```<language>
<panel type="info">
    <panel-title>Info Panel</panel-title>
    <panel-body>
        <p>
            Learn how to build ASP.NET Core apps that can run anywhere.
        </p>
    </panel-body>
</panel>
```

End Result:

```<language>
<div class="panel panel-info">
	<div class="panel-heading">
		<h3 class="panel-title">Info Panel</h3>
       </div>
	<div class="panel-body">
		<p>
			Learn how to build ASP.NET Core apps that can run anywhere.
		</p>
	</div>
</div>
```

**Comment by dpaquette** [comment]
Yes, please do submit a pull request!

**Comment by dpaquette** [comment]
Closed as part of PR #46
